ovs-integration-tests/src/main/java/org/dcsa/ovs/itests/config/TestConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.dcsa.ovs.itests.config;
+
+import com.github.fge.jsonschema.cfg.ValidationConfiguration;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+import org.hamcrest.Matcher;
+
+import static com.github.fge.jsonschema.SchemaVersion.DRAFTV4;
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchemaInClasspath;
+
+public class TestConfig {
```

Comment:
That is a misleading and/or confusing name. Could you name it SchemaValidator or something less confusing instead.

Reply:
Also put it in a different package please, schemavalidation has nothing to do with config